package.json

```diff
@@ -14,7 +14,7 @@
   "dependencies": {
     "@emotion/react": "~11.10.5",
     "@emotion/styled": "~11.10.5",
-    "@graasp/apps-query-client": "github:graasp/graasp-apps-query-client#70-errors-mockApi",
+    "@graasp/apps-query-client": "github:graasp/graasp-apps-query-client",
```

Comment:
If you want you can point to the npm package. But you may have to release a new version [here](https://github.com/graasp/graasp-apps-query-client/pull/72)

Reply:
Okay, all new releases are pushed to npm directly?

Reply:
Yes they are published to npm as soon as release-please creates a new release.